modules/data-loading/pages/load-from-spark-dataframe.adoc

```diff
@@ -88,6 +92,7 @@ To let TigerGraph parse the data correctly, please make sure you are setting `lo
 include::partial$spark/pre_def_options.adoc[]
 
 [NOTE]
+.Authentication
 ====
 Authenticate with the username/password or secret to support token refresh, otherwise, make sure the lifetime of the token is long enough for the loading job.
```

Comment:
```suggestion
Authenticate with the username/password or secret to automatically maintain a valid token; otherwise, make sure the lifetime of your token is long enough for the loading job.
```